app/liquid_tags/organization_tag.rb

```diff
@@ -22,7 +23,16 @@ def render(_context)
   end
 
   def parse_slug_to_organization(organization)
-    organization = Organization.find_by(slug: organization)
+    forem_domain = URL.url
+    if organization.starts_with?(forem_domain)
```

Comment:
I always get this method name wrong! 

https://ruby-doc.org/core-3.1.0/String.html#method-i-start_with-3F

Sorry for the misleading suggestion